ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.ts

```diff
@@ -91,9 +84,13 @@ export class ActivitiPeople {
     }
 
     involveUser(user: User) {
+        this.showAssignment = false;
+        if (user === undefined || user === null) {
+            return;
+        }
         this.peopleService.involveUserWithTask(this.taskId, user.id.toString())
             .subscribe(() => {
-                this.people.push(user);
+                this.people = [...this.people, user];
```

Comment:
ya I tried without that. Its not working with "this.people.push(user);" thatswhy changed to "this.people = [...this.people, user];". may be its issue at data-table end.